firmware/init/sensor/init_vehicle_speed_sensor.cpp

```diff
@@ -0,0 +1,26 @@
+#include "pch.h"
+
+#include "init.h"
+#include "vehicle_speed.h"
+#include "function_pointer_sensor.h"
+
+struct GetVehicleSpeedWrapper {
+	DECLARE_ENGINE_PTR;
+
+	float getVehicleSpeed() {
+		return ::getVehicleSpeed(PASS_ENGINE_PARAMETER_SIGNATURE);
+	}
+};
+
+static GetVehicleSpeedWrapper vehicleSpeedWrapper;
+
+static FunctionPointerSensor vehicleSpeedSensor(SensorType::VehicleSpeed,
+[]() {
+	return vehicleSpeedWrapper.getVehicleSpeed();
+});
+
+void initVehicleSpeedSensor(DECLARE_ENGINE_PARAMETER_SIGNATURE) {
```

Comment:
I don't see a call to `initVehicleSpeedSensor`, so the sensor never gets registered!

Reply:
@alxrMironov ^^^ this one is probably the most important one to get this PR less red more green?